### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by tbaxter-18f**

### CODE REVIEW

**Review by Sgtpluck — APPROVED** [review]

**Comment by mgwalker** [comment]
This probably warrants a minor version bump in the `package.json` so it can be published to npm. But I have no idea. Who are you people?

**Comment by Sgtpluck** [comment]
hmm should we wait until it's officially signed to merge?

**Comment by tbaxter-18f** [comment]
I don't think so. It's happening. Better to have the notification mechanisms in place, especially given everybody scrambling today.

**Pull request merged**
